### ISSUE TITLE
Not compiling on Ubuntu 20 (Rust 1.43.0)

### ISSUE THREAD

**Issue by Skydev0h** [tonlabs/TVM-linker#206]
I cannot compile TVM Linker on Ubuntu 20 with freshly installed Cargo and Rust.
The problem seems to relate to TON SDK v0.26.0 judging from the error, but it happens when compiling _this_ project.

![image](https://user-images.githubusercontent.com/5331782/98518040-80114480-2277-11eb-8c29-545d7187b34d.png)

```
   Compiling ton_sdk v0.26.0 (https://github.com/tonlabs/ton-sdk.git#b66b0e78)
error[E0277]: the trait bound `std::vec::Vec<api_info::Field>: std::convert::From<[api_info::Field; 6]>` is not satisfied
   --> /home/skydev/.cargo/git/checkouts/ton-sdk-11823a0ea1fdf592/b66b0e7/ton_sdk/src/transaction.rs:162:34
    |
162 | #[derive(Serialize, Deserialize, ApiType, Debug, PartialEq, Clone, Default)]
    |                                  ^^^^^^^ the trait `std::convert::From<[api_info::Field; 6]>` is not implemented for `std::vec::Vec<api_info::Field>`
    |
    = help: the following implementations were found:
              <std::vec::Vec<T> as std::convert::From<&[T]>>
              <std::vec::Vec<T> as std::convert::From<&mut [T]>>
              <std::vec::Vec<T> as std::convert::From<std::borrow::Cow<'a, [T]>>>
              <std::vec::Vec<T> as std::convert::From<std::boxed::Box<[T]>>>
            and 5 others
    = note: required because of the requirements on the impl of `std::convert::Into<std::vec::Vec<api_info::Field>>` for `[api_info::Field; 6]`
    = note: this error originates in a derive macro (in Nightly builds, run with -Z macro-backtrace for more info)

error: aborting due to previous error

For more information about this error, try `rustc --explain E0277`.
error: could not compile `ton_sdk`.
```

**Comment by Skydev0h** [comment]
It seems the prerequisite `Rust v. 1.38 or newer` is incorrect.
Using `v1.43` from ubuntu repository does not work.
If I use `rustup` and install `v1.47` rust, it works.
Please verify the version in prerequisites, it may be wrong.

**Comment by joydark** [comment]
Fixed in 562b2f29d7170432e3456ac3110cc59994f02190
